sdk/src/main/java/ly/count/android/sdk/ModuleRequestQueue.java

```diff
@@ -300,5 +340,15 @@ public void eraseWrongAppKeyRequests() {
                 requestQueueEraseAppKeysRequestsInternal();
             }
         }
+
+        /**
+         * Send request data after removing the predefined keys
```

Comment:
Add comment that they should be super sure about consent. 
Say which consent checking is being performed here and that they should perform additional checks themselves.

Reply:
Say that each key and value will be http url encoded